### ISSUE TITLE
Fine tuning SDXL inpainting

### ISSUE THREAD

**Issue by EnricoBeltramo** [huggingface/diffusers#4680]
**Is your feature request related to a problem? Please describe.**
Training with LORA of SDXL refiner for inpainting

**Describe the solution you'd like**
Is possibile to modify the solution proposed in: https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/README_sdxl.md 
in order to train refiners model too (i.e. for fine tune inpainting models)?

**Comment by sayakpaul** [comment]
How common is fine-tuning the refiner? I am yet to see some promising results from that.

**Comment by williamberman** [comment]
Yep! inpainting for sdxl is a wip here https://github.com/huggingface/diffusers/pull/4746

**Comment by abdellah-lamrani-alaoui** [comment]
Thanks a lot for the PR, will be super useful.
Quick question, do you plan on training and releasing a SDXL model fine-tuned for the inpainting task?

**Comment by williamberman** [comment]
Yeah we plan on releasing an inpainting fine-tuned sdxl :) The current objective is more to have a working checkpoint than a reproducible training script

**Comment by abdellah-lamrani-alaoui** [comment]
That sounds great!
Do you have an ETA for releasing the model?

**Comment by williamberman** [comment]
@abdellah-lamrani-alaoui https://huggingface.co/diffusers/stable-diffusion-xl-1.0-inpainting-0.1 !

**Comment by GitHub1712** [comment]
could we train this ourself?

**Comment by GitHub1712** [comment]
> @abdellah-lamrani-alaoui https://huggingface.co/diffusers/stable-diffusion-xl-1.0-inpainting-0.1 !

How did you made it? Is there any training code or example?

**Comment by lvZic** [comment]
> How did you made it? Is there any training code or example?

How did you made it? Is there any training code or example?

**Comment by lawsonxwl** [comment]
> @abdellah-lamrani-alaoui https://huggingface.co/diffusers/stable-diffusion-xl-1.0-inpainting-0.1 !

could you release the training code?

**Comment by crapthings** [comment]
Are there any scripts available for training today?

**Comment by sayakpaul** [comment]
https://github.com/huggingface/diffusers/pull/6922

**Comment by crapthings** [comment]
> #6922

This looks really good! Can it be used to train outpainting? For example, inverting the mask or masking out the entire background to keep only certain content.

**Comment by tingxueronghua** [comment]
> #6922

Could this reproduce similar performances?

**Comment by tingxueronghua** [comment]
> #6922

And is there any scripts for sd-xl-inpainting?

**Comment by sayakpaul** [comment]
Cc: @patil-suraj

**Comment by hzphzp** [comment]
same question
